deployment/kube-batch/templates/scheduling_v1alpha1_queue.yaml

```diff
@@ -2,6 +2,8 @@ apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
   name: queues.scheduling.incubator.k8s.io
+  annotations:
+    "helm.sh/hook": "crd-install"
```

Comment:
can you also help to add this hook for `PodGroup`?

Reply:
The YAML files in the chart do not use `PodGroup`, so it seems unnecessary? I'll add it if needed :)

Reply:
Added it in case that some logic(e.g. logic in the `kube-batch` deployment) depends on it.

Reply:
thanks :) Just in case we add some other dependency on that.